esmvaltool/cmorizers/data/formatters/datasets/hadcrut5.py

```diff
@@ -27,7 +27,7 @@
 import numpy as np
```

Comment:
This script is currently not working because of the call to the cmorization function.